src/app/components/queue/CreateQuestion.tsx

```diff
@@ -0,0 +1,47 @@
+import { Box, Button } from "@mui/material";
+import { FaArrowRight } from "react-icons/fa";
```

Comment:
You can get the same arrow from MUI icon library. I would say we should use that since that's what the Figma is based off.